example.js

```diff
@@ -73,6 +73,18 @@ const { prompt } = require('./');
                 { title: 'Gibson' },
                 { title: 'Grant' },
             ]
+        },
+        {
+            type: 'number',
+            name: 'prompt',
+            message: 'This will be overridden',
+            onRender() {
+                this.no = (this.no || 0) + 1;
+                this.msg = `Enter a number (e.g. ${this.no})`;
+                new Promise(resolve => setTimeout(resolve, 1000)).then(() => {
+                    if (!this.closed) this.render();
```

Comment:
I wonder if this check should be built in?
I forgot about it and the prompt rendered after completion/cancellation:
<img width="345" alt="screen shot 2018-10-27 at 12 42 44" src="https://user-images.githubusercontent.com/2302254/47603012-6b421b00-d9e6-11e8-9360-c145a13b3636.png">
<img width="322" alt="screen shot 2018-10-27 at 12 44 47" src="https://user-images.githubusercontent.com/2302254/47603013-6b421b00-d9e6-11e8-96bc-0f7ca5ad7b8a.png">